src/app/app.component.html

```diff
@@ -30,21 +30,11 @@
 		<form onsubmit="return false">
 			<!-- auto complete -->
 			<app-autocomplete
+				(selectedPackage)="onPackageNameSubmit()"
 				[autocomplateOptions]="autocompleteOptions$ | async"
 				[formControl]="addPackage"
 				[loadedEntities$]="packageNames.observable$"
 			></app-autocomplete>
-
-			<!-- add button -->
-			<button
-				mat-raised-button
-				type="submit"
-				color="primary"
-				[disabled]="!addPackage.value || addPackage.invalid"
-				(click)="onPackageNameSubmit()"
-			>
-				Add
-			</button>
```

Comment:
It is fine to remove `[disabled]="!addPackage.value || addPackage.invalid"` because we already have `	readonly packageErrorsHandler = this.errorHandlerService.getInputErrorsHandler('package name');` inside `AutocompleteComponent` so event if you manually type an already selected package name into the search bar, it will stop you with an error "package name is a duplicate"

Reply:
yea this would only be an issue if we had the button, thanks for summarizing code change